### ISSUE TITLE
windows clearing clipboard via osc fails

### ISSUE THREAD

**Issue by theimpostor** [alacritty/alacritty#8319]
Using alacritty on windows, while ssh'd into a linux machine, if I print the OSC 52 sequence to copy the empty string to the clipboard, alacritty logs a warning and the clipboard is not changed.

```
❯ echo -e "\x1b]52;c;\a"
```

```
[144.139578500s] [WARN ] [alacritty] Unable to store text in clipboard: The parameter is incorrect. (os error 87)
```

The same sequence works in alacritty in macos (no warnings, clipboard is cleared).

### System

OS: Windows 10
Version: alacritty 0.14.0-rc2 (7552ae7)

**Comment by theimpostor** [comment]
Thanks. FYI it fails with `\e\\` as well:

```
❯ echo -e "\e]52;c;\e\\"
```

**Comment by chrisduerr** [comment]
It's possible Windows messes things up. Both work on Linux.

**Comment by chrisduerr** [comment]
Actually looking closer into it, the error is at a pretty late stage when we're writing a string to the clipboard:

https://github.com/alacritty/alacritty/blob/master/alacritty/src/clipboard.rs#L64-L66

The only failure case here I could see is if Windows' API doesn't accept an empty string, which seems a bit nonsensical to me. The crate we're using on Windows also doesn't have a function to explicitly clear the clipboard, so generally speaking that should be working.

I'd assume using `echo -e "\e]52;c; \e\\"` works? So just writing anything but an empty string?

**Comment by shoda-tibco** [comment]
Yes doing the echo with actual base64 encoded data works to populate the clipboard with alacritty on Windows.

**Comment by chrisduerr** [comment]
> Yes doing the echo with actual base64 encoded data works to populate the clipboard with alacritty on Windows.

Yeah the base64 decryption isn't going to be the problem, since we reach the part where we pass a string to the Windows clipboard. And that string is guaranteed to be valid UTF-8. So I can only assume that the Windows clipboard crate does not support an empty string as payload.

I'll reopen the issue for now, unfortunately there's no Windows developers. However I'd imagine it should be pretty simple to fix for anyone interested in actually looking into it.

**Comment by j4james** [comment]
FYI, it was previously mentioned in https://github.com/alacritty/copypasta/issues/65 that the v3 version of clipboard-win doesn't allow you to set an empty string to the clipboard, so I'm assuming that's why this isn't working. This was supposedly fixed in v4.4, but the PR to upgrade seems to have been rejected for some reason.

**Comment by kchibisov** [comment]
it's because v5x was available and there were no clear indication on the PR why v4x was needed, if it fixes the bug, then it's fine to re-open to fix this.

**Comment by wells2kb** [comment]
![Image](https://github.com/user-attachments/assets/2fbff558-f8c8-4c03-8626-d56d8ed00c49)
I fixed by updating to 5.4.0

**Comment by wells2kb** [comment]
https://github.com/alacritty/copypasta/pull/78
